de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/provider/RequirementsAttributeLabelProvider.java

```diff
@@ -94,23 +96,33 @@ public String getColumnText(Object object, int columnIndex) {
 
 				return valueSwitch.getValueString(propertyInstance);
 
+			} else if (columnIndex == VERIFICATION_COLUMN) {
+
+				APropertyInstance propertyInstance = ca.getPropertyInstances().get(REQUIREMENT_VERIFICIATION_PROPERTY_NUMBER);
+				redirectNotification(propertyInstance, object);
+				ATypeInstance ti = valueSwitch.doSwitch(propertyInstance);
+				redirectNotification(ti, object);
+				
+				return getVerificationLabel(new Requirement(ca));
+				
+
 			} else if (columnIndex == TRACE_COLUMN) {
 
 				APropertyInstance propertyInstance = ca.getPropertyInstances().get(REQUIREMENT_TRACE_PROPERTY_NUMBER);
 				redirectNotification(propertyInstance, object);
 				ATypeInstance ti = valueSwitch.doSwitch(propertyInstance);
 				redirectNotification(ti, object);
 				
-				return getVerificationLabel(new Requirement(ca));
+				return getTracingLabel(new Requirement(ca));
 				
 
-			} else if (columnIndex > STATUS_COLUMN) {
+			} else if (columnIndex >= UiSnippetCustomRequirementsAttributeTable.FIXED_COLUMNS_NUMBER) {
 				APropertyInstance propertyInstance = ca.getPropertyInstances()

```

Comment:
APropertyInstance propertyInstance = ca.getPropertyInstances().get(REQUIREMENT_TRACE_PROPERTY_NUMBER); Duplicate in else if

Reply:
Well, its always different properties with differnt XXX_PROPERTY_NUMBER. I moved the two lines in between to a new method now.